src/components/ProfileHeader/index.js

```diff
@@ -0,0 +1,2 @@
+import ProfileHeader from "./ProfileHeader";
+export default ProfileHeader;
```

Comment:
Let's add an empty line between the statements and one at the end of the file
```suggestion
import ProfileHeader from "./ProfileHeader";

export default ProfileHeader;

```